src/driverApp/tests/driver.test.js

```diff
@@ -0,0 +1,104 @@
+import bcrypt from 'bcrypt';
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app.js';
+import logger from '../../configs/winston.js';
+import DataSource from '../../data-source.js';
+import User from '../../models/user.js';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('driver router tests', () => {
+	before(async () => {
+		try {
+			await DataSource.query('TRUNCATE "Driver" CASCADE');
+			await DataSource.query('TRUNCATE "User" CASCADE');
+		} catch (error) {
+			logger.error(error.message);
+		}
+	});
+	let token;
+	let createdDriverId;
+	let driverInfo;
+	it('should login an admin user', async () => {
+		const password = await bcrypt.hash('Andela@1234', 5);
+		const newUser = User.create({
+			email: 'test123@gmail.com',
+			password,
+			firstName: 'test',
+			lastName: 'Andela',
+			role: 'admin',
+		});
+		const user = await newUser.save();
+		const info = { email: user.email, password: 'Andela@1234' };
+		const response = await chai
+			.request(app)
+			.post('/api/v1/accounts/login')
+			.send(info);
+		expect(response).to.have.status(200);
+		token = response.body.data.access_token;
+	});
+	it('should get all drivers', async () => {
+		const response = await chai
+			.request(app)
+			.get('/api/v1/drivers')
+			.set('Authorization', `Bearer ${token}`);
+		expect(response).to.have.status(200);
+	});
```

Comment:
can you expect to get a response with no drivers "if possible", then after creating a new one, this is where u can expect to get actual drivers....

Reply:
Thanks a lot! I forgot about that